ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-templates.yml

```diff
@@ -30,12 +30,8 @@ default_templates:
           strip_empty_keys: false
 
     ip_addressing:
-      router_id: 'ip_addressing/router-id.j2'
-      ipv6_router_id: 'ip_addressing/router-id-ipv6.j2'
-      mlag_ip_primary: 'ip_addressing/mlag-ip-primary.j2'
-      mlag_ip_secondary: 'ip_addressing/mlag-ip-secondary.j2'
-      mlag_l3_ip_primary: 'ip_addressing/mlag-l3-ip-primary.j2'
-      mlag_l3_ip_secondary: 'ip_addressing/mlag-l3-ip-secondary.j2'
+      python_module: 'ansible_collections.arista.avd.roles.eos_designs.python_modules.ip_addressing'
```

Comment:
Given that we have default variables in the classes for python modules and classes - do we actually need this?

Reply:
No we don't. Just trying to help someone who wants to change it.